code/game/objects/effects/anomalies.dm

```diff
@@ -60,6 +60,15 @@
 			step_towards(O,src)
 	for(var/mob/living/M in orange(4, src))
 		step_towards(M,src)
+	for(var/obj/O in orange(1,src))
+		if(!O.anchored)
+			var/mob/living/target = locate() in view(10,src)
+			if(!target)
+				return
+			else
+				O.throw_at(target, 5, 10)
+		return

```

Comment:
did you mean to ```return``` here?

this code will only ever throw one object at one mob.
and if the object is anchored it wont throw any at all.